### DISCUSSION TITLE
HTML5 video, video, divider, codesnippet

### DISCUSSION THREAD

**Discussion by bard-hobson** [react-page/react-page#1084] [Q&A]
Trying to use the builtin plugins (side question cannot find codesnippet) and while I see them in editor they do not show up in preview mode. Image and Text show up fine but the html5video, video, space, and divider do not. The code shows the div but nothing is in the data.

Created a custom image plugin that worked as well. 

import video from '@react-page/plugins-video';
import spacer from '@react-page/plugins-spacer';
import divider from '@react-page/plugins-divider';
import html5video from '@react-page/plugins-html5-video';

const cellPlugins = [slate(), divider, spacer, video, html5video, customImage, postEntryInfo];

**Comment by bard-hobson** [comment]
[FIXED]

I am an idiot. I was using it on another page and did not import the other plugins. Make sure to do that. Only thing still missing is codesnippet. Could not find import.

Had:
const cellPlugins = [slate(), image];

Needed:
const cellPlugins = [slate(), image, postEntryInfo, customImage, divider, spacer, video, html5video, customVideo];

**Reply by macrozone** [reply]
code-snippet only exists as an example, its rather short: https://github.com/react-page/react-page/blob/master/examples/plugins/codeSnippet.tsx
